roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
```

Comment:
It might be argsparse magic.. But https://github.com/ryshah/swifttool/blob/master/swifttool/client.py#L310 user here is none if I don't specify -u option with swifttool